### PR TITLE
test: improved local test running

### PR DESCRIPTION
**Pull request by kirrg001**
- Added node script to add terminal aliases
- Integrate `mocha --watch`
- Removed redundant sh helper scripts. Only one sh helper script `bin/run-tests.sh`.

See CONTRIBUTING.md.

### CODE REVIEW

**Comment by kirrg001** [comment]
There is one problem with the cleanup of our global agent and `--watch` in our test env.

**Comment by kirrg001** [comment]
refs https://github.com/mochajs/mocha/issues/5149

**Comment by kirrg001** [comment]
Fixed https://github.com/instana/nodejs/pull/1441/commits/586cd0a33fd9d7823edc0ba12fe2be73d8772651

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,15 +26,33 @@ Install the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting
...
 node bin/start-test-containers.js --mongo --redis
 ```
 
-It's not recommended to run all tests using `bin/run-tests.sh` - it takes too long. Take a look at the root package.json and run a specific test group or run e.g. `bin/run-collector-tests.sh` with the mocha `.only` attribute.
+### Using terminal aliases
+
+Add aliases to your terminal:
+
+```sh
+node bin/add-test-aliases.js bash|zsh
```
Comment (aryamohanan) [inline CONTRIBUTING.md:40]: `node bin/add-test-aliases.js bash` worked for me.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,15 +26,33 @@ Install the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting
...
+```
+
+Add a mocha `.only` on the test you would like to execute and then run the target scope:
+
+```sh
+runcollector
```
Comment (aryamohanan) [inline CONTRIBUTING.md:46]: _nitpick:_
```suggestion
runcollector        # Run the 'collector' scope with watch mode
runcollector-nw     # Run the 'collector' scope without watch mode
```

**Review by aryamohanan — APPROVED** [review]
LGTM.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,15 +26,34 @@ Install the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting
...
+```sh
+node bin/add-test-aliases.js bash|zsh
+```
+
+Add a mocha `.only` on the test you would like to execute and then run the target scope:
+
+```sh
+runcollector        # Run the 'collector' scope with watch mode
+runcollector-nw     # Run the 'collector' scope without watch mode
+runcollector-nw (no watch)
```
Comment (kirrg001) [inline CONTRIBUTING.md:48]:
```suggestion
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (abhilash-sivan) [reply CONTRIBUTING.md:40]: I verified in zsh.

`node bin/add-test-aliases.js zsh` worked fine for me!

**Review by abhilash-sivan — APPROVED** [review]

**Pull request merged**